llpc/lower/llpcSpirvLowerRayTracing.cpp

```diff
@@ -1305,8 +1367,11 @@ Value *SpirvLowerRayTracing::getShaderIdentifier(ShaderStage stage, Value *shade
 void SpirvLowerRayTracing::createAnyHitFunc(Value *shaderIdentifier, Value *shaderRecordIndex) {
   Function *func = dyn_cast_or_null<Function>(m_module->getFunction(RtName::CallAnyHitShader));
   if (!func) {
-    auto funcTy =
-        FunctionType::get(m_builder->getVoidTy(), {shaderIdentifier->getType(), shaderRecordIndex->getType()}, false);
+    SmallVector<Type *> tys = {shaderIdentifier->getType(), shaderRecordIndex->getType()};
+    for (unsigned i = 0; i < TraceParam::Count; ++i)
+      tys.push_back(PointerType::get(m_traceParamsTys[i], SPIRAS_Private));
```

Comment:
Due to opaque pointers, this should just be `m_builder->getPtrTy(SPIRAS_Private)`